webapp/js/components/App.js

```diff
@@ -52,23 +55,26 @@ var App = React.createClass({
 
     return (
       <div>
-				<header className="banner global" role="banner">
-				  <nav role="navigation" className="nav-primary">
-				    <span id="main-navigation-link"><a href="#navigation">Jump to site nav</a></span>
-				    <div className="logo">
-				      <a className="logo-ubuntu" href="/">
-				        <img width="106" height="25" src={LOGO} alt="" />
-				        <span>{M({id:"title"})}</span>
-				      </a>
-				    </div>
-						<div>
-							<form id="language-form" className="header-search">
-									{/* Add more languages here */}
-									{this.renderLanguage('en')}
-									{/* this.renderLanguage('zh') */}
-							</form>
+				<header className="p-navigation" role="banner">
+					<div className="row">
+						<div className="p-navigation__logo">
+								<div className="nav_logo">
+										<img src="/static/images/logo-ubuntu-white.svg" alt="Ubuntu" />
+										<span>{M({id:"title"})}</span>
+								</div>
 						</div>
-				  </nav>
+
+						<nav role="navigation" className="p-navigation__nav">
+							<span className="u-off-screen"><a href="#navigation">Jump to site nav</a></span>
+							<Navigation />
+								{/*<form id="language-form" className="header-search">*/}
```

Comment:
We only have an English translation at the moment, so I've been wanting to remove the option for a while. If/when we get the Chinese translation done, we can re-enable this - all the other internationalization still works.